src/styles/main.scss

```diff
@@ -1,3 +1,247 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+ul,
+p,
+h2 {
   margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style: none;
+}
+
+/*Variables*/
```

Comment:
create separate files for reset, variables and other scss utils and bem blocks
<img width="650" alt="image" src="https://user-images.githubusercontent.com/102411130/223358818-7601779c-b1a5-4006-bb6d-6b1530406143.png">